### PR TITLE
HPCC-14504 Expose Thor crc read+write options

### PR DESCRIPTION
**Pull request by jakesmith**
Signed-off-by: Jake Smith jake.smith@lexisnexis.com

### CODE REVIEW

**Comment by jakesmith** [comment]
@ghalliday - please review

File: thorlcr/graph/thgraph.hpp [hunk 1]
```diff
@@ -1045,12 +1051,12 @@ class graph_decl CActivityBase : public CInterface, implements IExceptionHandler
...
-    StringBuffer &getOpt(const char *prop, StringBuffer &out) const;
-    bool getOptBool(const char *prop, bool defVal=false) const;
-    int getOptInt(const char *prop, int defVal=0) const;
-    unsigned getOptUInt(const char *prop, unsigned defVal=0) const { return (unsigned)getOptInt(prop, defVal); }
-    __int64 getOptInt64(const char *prop, __int64 defVal=0) const;
-    unsigned __int64 getOptUInt64(const char *prop, unsigned __int64 defVal=0) const { return (unsigned __int64)getOptInt64(prop, defVal); }
+    StringBuffer &getOpt(const char *prop, StringBuffer &out) const { return container.getOpt(prop, out); }
+    bool getOptBool(const char *prop, bool defVal=false) const { return container.getOptBool(prop, defVal); }
+    int getOptInt(const char *prop, int defVal=0) const { return container.getOptInt(prop, defVal); }
+    unsigned getOptUInt(const char *prop, unsigned defVal=0) const { return (unsigned)container.getOptInt(prop, defVal); }
```
Comment (ghalliday) [inline thorlcr/graph/thgraph.hpp]: minor: Should really call container.getOptUInt, and allow that to cast.  Same for getOptUInt64

**Comment by ghalliday** [comment]
@jakesmith one tiny comment, otherwise looks good.

File: thorlcr/graph/thgraph.hpp (hunk 1 above)
Comment (jakesmith) [inline thorlcr/graph/thgraph.hpp]: agree, will change.

**Comment by jakesmith** [comment]
>minor: Should really call container.getOptUInt, and allow that to cast. Same for getOptUInt64

Fixed and squashed
@ghalliday

**Pull request merged**